Integrations/src/main/java/io/deephaven/integrations/python/PythonUtilities.java

```diff
@@ -36,4 +36,29 @@ static PyObject pyListenerFunc(final PyObject pyObject) {
                     "class instance with an onUpdate method");
         }
     }
+
+    /**
+     * Creates a callable PyObject, either using method.apply() or __call__(), if the pyObjectIn has such methods available.
+     *
+     * @param pyObjectIn the python object providing the function - must either be callable or have an `apply` attribute
+     *                   which is callable.
+     * @return pyCallable that can be called directly with arguments using pyCallable.call(...).
+     */
+    static PyObject pyCallableGenerator(PyObject pyObjectIn) {
```

Comment:
1) rename to pyApplyFunc
2) the two functions in this class are nearly identical.  If you take the meat and create a function that looks like
` static PyObject pyCallable(final PyObject pyObject, final String pyAttribute)`, you can reduce all of the duplication.